includes/wc-template-functions.php

```diff
@@ -2456,7 +2456,7 @@ function wc_display_item_meta( $item, $args = array() ) {
 		) );
 
 		foreach ( $item->get_formatted_meta_data() as $meta_id => $meta ) {
-			$value = $args['autop'] ? wp_kses_post( $meta->display_value ) : wp_kses_post( make_clickable( trim( strip_tags( $meta->display_value ) ) ) );
+			$value = $args['autop'] ? wp_kses_post( $meta->display_value ) : wp_kses_post( make_clickable( trim( $meta->display_value ) ) );
```

Comment:
Curious why we would remove br tags? Have a customer asking in a ticket why the display is different between the "order received" page and the emails. They are just wanting the br tags to still be there.

In emails, it should be fine to allow for breaks, yea?